ddR/R/ddR.R

```diff
@@ -246,6 +255,8 @@ dmapply <- function(FUN ,..., MoreArgs=list(),
   # Ensure that ... arguments are of equal length. length() works correctly for data.frame,
   # arrays, and lists
   lens <- vapply(dargs,function(x) {
+    # TODO Clark: Why even allow the possibility of this happening where
```

Comment:
How would you prevent it from happening? If the user creates a dobject with one driver, e.g.:

`useBackend(parallel)`
`a <- dlapply(...)`
`useBackend(Spark)`
`b <- dlapply(...)`

We need to catch the error if/when the user does
`dmapply(function()...,a,b)`, since `a` and `b` came from different backends. All input objects need to have been derived from the same backend.